test/dotnet-new3.UnitTests/Sdk/SdkCommandSpec.cs

```diff
@@ -1,22 +1,30 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+#nullable enable
```

Comment:
These classes were copied from dotnet/sdk - just wondering if we want to keep modifying them - it makes them diverge from original and if we need to update them from dotnet/sdk we probably have spend time to merge.

Reply:
Sounds like a valid concern to me. I will revert this.